### ISSUE TITLE
AP is close to 0 for my trained model

### ISSUE THREAD

**Issue by zhangjing9701** [fundamentalvision/Deformable-DETR#55]
@jackroos  

Hi, about a month ago, I downloaded the provided codes and used the command "GPUS_PER_NODE=2 ./tools/run_dist_launch.sh 2 ./configs/r50_deformable_detr.sh" to run the code. The parameters setting is default except that the batch size is set to 1 due to memory issue. (I have modified batch size=1 as the default value in main.py)
However, when I evaluated the model saved at epoch 48 using the command "./configs/r50_deformable_detr.sh --resume exps/r50_deformable_detr/checkpoint.pth --eval", the AP is close to 0. 
![image](https://user-images.githubusercontent.com/48611303/110750225-20ef2b00-827d-11eb-857f-385a22ce4123.png)

Then, I downloaded the provided model "r50_deformable_detr-checkpoint.pth", the evaluation result is right.
![微信图片_20210311150112](https://user-images.githubusercontent.com/48611303/110749283-e8028680-827b-11eb-8e3b-2f92f4bf1008.png)

I wonder the reason of the abnormal AP for my trained model, and how to solve it? Thanks very much!!!

-------
a student from USTC
six department
zjing5@mail.ustc.edu.cn (Jing Zhang)

**Comment by 1224wxwx** [comment]
I met the same problem. Could you please tell me how to resolve it? @small-mushroom

**Comment by zhangjing9701** [comment]
> I met the same problem. Could you please tell me how to resolve it? @small-mushroom
When running the released code, I used 2 GPUs and met the above problem. I solved the problem by multiply the default learning rate by 0.1.

**Comment by mei123456789** [comment]
@zhangjing9701 Hello, did you use pre-training weights when you trained your data set? Looking forward to your reply. Thank you

**Comment by 1757525671** [comment]
> @zhangjing9701 Hello, did you use pre-training weights when you trained your data set? Looking forward to your reply. Thank you

I also encountered this problem, the reduction of learning rate will not solve it, very awful. I suspect something is wrong with the code.

**Comment by zhangjing9701** [comment]
> @zhangjing9701 Hello, did you use pre-training weights when you trained your data set? Looking forward to your reply. Thank you

I only modified the batch size to 1 and multiplied the default learning rate by 0.1, all other settings were default. You can refer to the released code.

**Comment by kobebrip** [comment]
我也是这个问题，但是只修改了批量大小为1，并将默认学习率乘以0.1，并没有解决问题
